admin/sections/config/form.tsx

```diff
@@ -41,6 +45,8 @@ export const ConfigForm = ({ setConfigured }: { setConfigured: any }) => {
       } else {
         setConfigured(false);
       }
+      // Run the post submit callbacks
```

Comment:
does it hurt so much if we just call `reloadDemoChat` here? would simplify everything

Reply:
(or add a guard and only calling it if the chat config was updated, see other comment)